rancher/rancher
# [2.7] Update the SUC version in the Dockerfile

## Issue: https://github.com/rancher/rancher/issues/40565
 
## Problem
Changes were made to the SUC chart allowing us to specify if we want to deploy PSP resources along with the chart. This was developed and tested locally, however the Dockerfile was never updated
 
## Solution
Bump the SUC version in the Dockerfile to match that of the latest SUC chart for 2.7, `102.0.0+up0.4.0`

## Testing
We want to run through the test steps for #40565, #40384 again

## Engineering Testing
### Manual Testing
We should do the above

### Automated Testing
n/a

## QA Testing Considerations
Make sure version `102.0.0+up0.4.0` is deployed and no PSP errors are seen when creating 1.25 clusters.

Make sure that `kubectl get managedcharts.management.cattle.io -A` does not show any errors. This step was omitted in prior testing and should be double checked upon retesting
 
### Regressions Considerations
n/a
